scripts/ProcessTemplates.ps1

```diff
@@ -118,11 +118,14 @@ Function AddCategory() {
     $categories.$categoryName.DescriptionByLanguage = @{ }
 
     $categorySettings | Get-Member -type NoteProperty | Foreach-Object {
-        $languageProperties = $categorySettings.($_.name)
+        # ignore the $schema field which is needed for localization of categoryResources.json
+        if ($_.name -ne '$schema') {

```

Comment:
doesn't this mean that any extra properties in this file are going to cause this to fail?  should this be simplified to just grab fields it knows about instead of ignoring one specific field?

Reply:
the only expected fields here are the language field, we can change that